MAUI/Barcode/getting-started.md

```diff
@@ -0,0 +1,167 @@
+
+{% highlight C# hl_lines="17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace GaugeMauiSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+
+## Adding a namespace
+
+Add the following namespace to add .NET MAUI Badge Notifications.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    xmlns:barcode="clr-namespace:Syncfusion.Maui.Barcode;assembly=Syncfusion.Maui.Barcode"
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+    using Syncfusion.Maui.Barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Initialize the SfBarcodeGenerator control
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<barcode:SfBarcodeGenerator Value="http://www.syncfusion.com" 
+                            HeightRequest="150"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+barcode.HeightRequest = 150;
+barcode.Value = "http://www.syncfusion.com";
+this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Intialize Barcode](images/getting-started/maui-intialize-barcode.png)
+
+N> The default symbology of SfBarcodeGenerator is `Code128`.
+
+## Initialize QR Code symbology
+
+You can set the required symbology type to the barcode generator based on input value by initializing the `Symbology` property. In the following code snippet, the QR code is set as the barcode symbology.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+        <barcode:SfBarcodeGenerator Value="https://www.syncfusion.com/" 
+                                    HeightRequest="350" 
+                                    WidthRequest="350">
+            <barcode:SfBarcodeGenerator.Symbology>
+                <barcode:QRCode />
+            </barcode:SfBarcodeGenerator.Symbology>
+        </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight c# hl_lines="5" %}
+
+SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+barcode.HeightRequest = 350;
+barcode.WidthRequest = 350;
+barcode.Value = "https://www.syncfusion.com/";
+barcode.Symbology = new QRCode();
+this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Intialize QR Code Symbology](images/getting-started/maui-symbology-qrcode.png)
+
+## Display input value
+
+The provided input value can be displayed below the barcode by enabling the `ShowText` property of barcode as like the following code snippet,
```

Comment:
The provided input value can be displayed below the barcode by enabling the `ShowText` property of barcode as shown in the following code sample.

Reply:
Changes updated.